src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx

```diff
@@ -336,6 +342,61 @@ export const DashboardPremiumUser: Story = {
             FreeBrokerScan: true,
             PremiumBrokerRemoval: true,
           }}
+          scanStatus="finished"
+        />
+      </Shell>
+    );
+  },
+};
+
+export const DashboardNoBreachesScanInProgress: Story = {
```

Comment:
Ooh, another permutation option... This is going to conflict with https://github.com/mozilla/blurts-server/pull/3423, where it would be implemented as another entry in `brokerOptions` :cry:

Reply:
Added `scan-in-progress` to the `brokerOptions` in https://github.com/mozilla/blurts-server/pull/3431/commits/745a73bbed958dd92fba3f43520c0c27622cc404.